src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -64,9 +67,30 @@
 from snowflake.connector.telemetry_oob import TelemetryService
 from snowflake.connector.time_util import DecorrelateJitterBackoff
 
+from .cache import SFDictCache
+
+OCSP_CACHE: SFDictCache[
+    tuple[bytes, bytes, int, str],
+    tuple[Exception | None, Certificate, Certificate, CertId, bytes],
+] = SFDictCache(
+    entry_lifetime=60 * 60 * 24,  # A day in seconds
```

Comment:
Similarly, use a macro. Maybe the macro could be put into `constants` so it can be shared.